### ISSUE TITLE
Cannot parse #if/#endif in enum

### ISSUE THREAD

**Issue by ChristopherRogers** [nicklockwood/SwiftFormat#281]
Running the following code through SwiftFormat (0.13.13)...

```swift
enum MyEnum {
    case copy

#if NEW_FEATURE_ENABLED
    case move
#endif
}
```

...gives the warning, "warning: unexpected token #endif at 6:0 in test.swift".

**Comment by nicklockwood** [comment]
@christopherrogers hmm, I thought this was fixed - I’ll take a look.

**Comment by nicklockwood** [comment]
@xuaninbox fixed in 0.34.0
